docs/using-add-on-modules.md

```diff
@@ -3,17 +3,25 @@ hide:
 - toc
 ---
 
-# Using Add-On Modules
+# Using add-on modules
 
-In order to reduce the number of external dependencies on the application being tested, Robolectric's shadows are split into various add-on packages. Only shadows for classes provided in the base Android SDK are provided by the main Robolectric module. Additional shadows for things like appcompat or the support library are provided by add-on modules. The table below lists the available add-on shadow packages:
+In order to reduce the number of external dependencies on the application being tested, Robolectric's shadows are split into various add-on packages. Only shadows for classes provided in the base Android SDK are provided by the main Robolectric module. Additional shadows are provided in dedicated packages. The table below lists the available add-on shadow packages:
 
-| SDK Package                          | Robolectric Add-On Package            |
-|--------------------------------------|---------------------------------------|
-| com.android.support.support-v4       | org.robolectric:shadows-supportv4    |
-| com.android.support.multidex         | org.robolectric:shadows-multidex      |
-| com.google.android.gms:play-services | org.robolectric:shadows-playservices |
-| org.apache.httpcomponents:httpclient | org.robolectric:shadows-httpclient    |
+| SDK package | Robolectric add-on package |
+|-----|-----|
+| `com.google.android.gms:play-services` | [`org.robolectric:shadows-playservices`](https://github.com/robolectric/robolectric/tree/robolectric-4.13/shadows/playservices) |
+| `org.apache.httpcomponents:httpclient` | [`org.robolectric:shadows-httpclient`](https://github.com/robolectric/robolectric/tree/robolectric-4.13/shadows/httpclient) |
 
-The above artifact names are in use since [Robolectric 3.5](https://github.com/robolectric/robolectric/releases/tag/robolectric-3.5). [Robolectric 3.4](https://github.com/robolectric/robolectric/releases/tag/robolectric-3.4) drops `shadows-` prefix from artifact names. Prior to 3.4, all artifact names are the same as latest, except that `shadows-supportv4` and `shadow-playservices` are `shadows-support-v4` and `shadow-play-services` respectively.
+> [!NOTE]
+> - [Robolectric 3.4](https://github.com/robolectric/robolectric/releases/tag/robolectric-3.4) doesn't include the `shadows-` prefix in the package name (i.e., `org.robolectric:playservices` and `org.robolectric:httpclient`).
+> - Before Robolectric 3.4, `org.robolectric:playservices` was named `shadow-play-services`.
 
-Note that `org.robolectric:shadows-supportv4` was deprecated at [Robolectric 4.8](https://github.com/robolectric/robolectric/releases/tag/robolectric-4.8), and was removed at [Robolectric 4.9](https://github.com/robolectric/robolectric/releases/tag/robolectric-4.9).
+## Legacy packages
```

Comment:
I can do that, yes.

Reply:
I've reworked the structure of the page:
- There are three sections: `Supported packages`, `Deprecated packages`, and `Removed packages`.
- The first two tables have a `Javadoc` column.
- The `shadows-httpclient` to the `Deprecated packages` section, since it is [deprecated](https://robolectric.org/javadoc/latest/org/robolectric/shadows/httpclient/package-summary.html).

I've updated the screenshot in the PR description with the result.